src/app/features/edit-nonce/edit-nonce.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+
+import { Box, BoxProps, color } from '@stacks/ui';
+
+import { SpaceBetween } from '@app/components/space-between';
+import { Caption } from '@app/components/typography';
+
+import { EditNonceDrawer } from './components/edit-nonce-drawer';
+
+export function EditNonce({ ...props }: BoxProps) {
+  const [isShowingEditNonce, setIsShowingEditNonce] = useState(false);
+
+  return (
+    <Box {...props}>
+      <SpaceBetween>
+        <Caption
+          _hover={{ cursor: 'pointer', textDecoration: 'underline' }}
+          color={color('brand')}
+          onClick={() => setIsShowingEditNonce(!isShowingEditNonce)}
+        >
+          Edit nonce
+        </Caption>
```

Comment:
click event attached to non-button/a tag element

Reply:
Recommend taking a look at https://github.com/jsx-eslint/eslint-plugin-jsx-a11y. I don't think it'd be easy to set up this linter, but reading the rules shows some of the most common accessibility mistakes

Reply:
Ok, thx, this was existing code but I'll look at refactoring it.